content/en/docs/contributing/e2e.md

```diff
@@ -30,18 +30,52 @@ run the tests:
   not required. For non-Linux hosts (i.e. OSX), you will need to ensure you have
   a relatively new version of `kubectl` available on your PATH.
 
+- `kind`: We use kind to provision a Kubernetes cluster.
+
 - An internet connection: tests require access to DNS, and optionally CloudFlare
   APIs (if a CloudFlare API token is provided).
 
 Bazel, Docker and `kubectl` should be installed through your preferred means.
 
+## Set up End-to-End Tests
+
+You need to have a Kind cluster running, if you don't have one set up you can set one up using:
```

Comment:
May be worth noting here that you MUST create the cluster using `./devel/cluster/create.sh` else the ACME HTTP01 end-to-end tests will fail, as they require the 'service cidr' to be set to `10.0.0.0/16` as the ingress controller is deployed with the fixed IP `10.0.0.15` to allow Pebble to access it on a predictable address for e2e tests (`certmanager.kubernetes.network` points to `10.0.0.15`, and that is the DNS name we use for validation during test runs...)

Reply:
Added a note!